site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -1,152 +1,137 @@
+In the above _only_ namespaces labeled as `development` will be able to attach
+their routes to the `Gateway`. The namespaces themselves must be decorated with
+the corresponding labels:
 
 ```yaml
-{% include 'v1alpha1/cross-namespace-routing/store-route.yaml' %}
+{% include 'v1alpha2/cross-namespace-routing/0-namespaces.yaml' %}
 ```
 
-This Route has straightforward routing logic as it just matches for 
-`/store` traffic which it sends to the `store` Service. The following snippet 
-of the [`gateways` field](/v1alpha2/references/spec/#networking.x-k8s.io/v1alpha1.RouteGateways) 
-controls which Gateways this Route can bind to:
+## Route Attachment Examples
+
+The store team deploys their route for the `store` Service in the `store-ns`
+Namespace:
 
 ```yaml
-  gateways:
-    allow: FromList
-    gatewayRefs:
-    - name: shared-gateway
-      namespace: infra
+{% include 'v1alpha2/cross-namespace-routing/store-route.yaml' %}
 ```
 
-`gateways.allow` can be configured for Gateways in the same Namespace as the
-Route (the default), all Gateways, or a list of specific Gateways. In this
-example the store and site teams decide to reference a specific Gateway. This is
-the least permissive choice which ensures that other Gateways in the cluster
-(perhaps created in the future at some point) will not bind with these Routes.
-If cluster administrators have full control over how Gateways are deployed in a
-cluster then a more permissive binding option could be configured on Routes. The
-less permissive the Gateway selection is, the less that application owners need
-to know about which Gateways are deployed. 
+This Route has straightforward routing logic as it just matches for
+`/store` traffic which it sends to the `store` Service.
 
 The site team now deploys Routes for their applications. They deploy two
 HTTPRoutes into the `site-ns` Namespace:
 
-- The `home` HTTPRoute acts as a default routing rule, matching for all traffic 
-to `foo.example.com/*` not matched by an existing routing rule and sending it to 
+- The `home` HTTPRoute acts as a default routing rule, matching for all traffic
+to `foo.example.com/*` not matched by an existing routing rule and sending it to
 the `home` Service.
-- The `login` HTTPRoute  routes traffic for `foo.example.com/login` to 
-`service/login-v1` and `service/login-v2`. It uses weights to granularly 
-control traffic distribution between them. 
+- The `login` HTTPRoute  routes traffic for `foo.example.com/login` to
+`service/login-v1` and `service/login-v2`. It uses weights to granularly
+control traffic distribution between them.
 
-Both of these Routes use the same Gateway binding configuration which specifies
-`gateway/shared-gateway` in the `infra-ns` Namespace as the only Gateway that these
-Routes can bind with.
+Both of these Routes use the same Gateway attachment configuration which
+specifies `gateway/shared-gateway` in the `infra-ns` Namespace as the only
+Gateway that these Routes can attach with.
```

Comment:
```suggestion
Gateway that these Routes want to attach to.
```
The language that makes the most sense for me for Route attachment is that the Route "is requesting to attach to", or "wants to attach to" the Gateway. I know, it's a seemingly trivial thing to call out, but if we can use this consistently, we'll be able to shape how people think of this new process, and we only have one chance for a first impression.